lisa/runners/legacy_runner.py

```diff
@@ -309,16 +309,22 @@ def _set_result(
         ), f"result name '{result.name}' doesn't match parsed name '{parsed_name}'"
 
         raw_status = information.pop("status")
-        if raw_status in ["FAIL", "ABORT"]:
+        if raw_status in ["FAIL", "ABORTED"]:
```

Comment:
the ABORTED and SKIPPED changes can be in a separate commit. So the changes can be more clear.